package.json

```diff
@@ -1,79 +1,102 @@
 {
-    "name": "chooser",
-    "version": "1.0.0",
-    "description": "The new and improved Creative Commons License Chooser",
-    "author": "akmadian <akmadian@gmail.com>",
-    "private": false,
-    "repository": {
-        "type": "git",
-        "url": "git+https://github.com/creativecommons/chooser.git"
+  "name": "chooser",
+  "version": "1.0.0",
+  "private": false,
+  "description": "The new and improved Creative Commons License Chooser",
+  "author": "akmadian <akmadian@gmail.com>",
+  "scripts": {
+    "serve": "vue-cli-service serve",
+    "build": "vue-cli-service build && shx rm -rf ./docs && shx cp -r ./dist ./docs",
+    "test:unit": "vue-cli-service test:unit",
+    "test:e2e": "vue-cli-service test:e2e",
+    "lint": "vue-cli-service lint",
+    "i18n:report": "vue-cli-service i18n:report -v --src src/**/*.vue --locales src/locales/**.json -o output.json",
+    "lint:js": "vue-cli-service lint",
+    "stylelint": "stylelint \"**/*.{vue,css}\"",
```

Comment:
Can you make stylelint only lint the content of 'src' folder? Otherwise it also tries to lint 'coverage', 'dist', or any other folders we might add later.

Reply:
There's **ignoreFiles: ['docs/css/*']** property in stylelint.config.js which will prevent stylelint to lint this folder. So in future we can add other folder here. But still I'll make stylelint only lint the content of 'src' folder.

Reply:
Also does this mean eslint is also linting **dist/js/*** or vue-cli-service will handle it and will lint-staged lint 'dist' while committing or not?

Reply:
I @rohan-goel we don't want the `/dist` folder linted, only the `src/` files we write. I think @obulat's original approach to specify `src/` is still the best approach.

Reply:
@zackkrida I have add `"lint": "vue-cli-service lint && npx stylelint ./src/**/*.{vue,css} --fix"` command in package.json. This will only lint src folder.